### PR TITLE
Implement sUSD Bridge

### PR DESCRIPTION
**Pull request by duyentb95**
Implement a native bridge on the main Synthetix page https://staking.synthetix.io/bridge
Following the steps in this blog: https://blog.synthetix.io/susd-bridge-enabled/

### CODE REVIEW

**Comment by duyentb95** [comment]
@jmzwar @bachstatter 
Have a great new week! Can you help me review this PR?

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```
Comment (jmzwar) [inline .idea/.gitignore:3]: Can remove these files

**Review by jmzwar — CHANGES_REQUESTED** [review]
We can remove the .idea folder

File: .idea/.gitignore (hunk 1 above)
Comment (duyentb95) [reply .idea/.gitignore:3]: Sure! I fixed it ^^
image

**Review by jmzwar — APPROVED** [review]

**Pull request merged**